annotate.py

```diff
@@ -128,20 +166,26 @@ def annotate_img(img_path, initials, size1=1, size2=1) :
                     print("Recording ", writepmtID+"-"+writestartingVal, ix, "x", iy,"y\n")
                     print("Record another selected point by pressing r.\nPress x to skip recording a number.\nPress f to finish recording features for this PMT.")
```

Comment:
Update screen instructions

Reply:
updated with increment and decrement keys.